bitnami/mongodb/templates/statefulset-arbiter-rs.yaml

```diff
@@ -58,9 +58,9 @@ spec:
       affinity:
 {{ toYaml .Values.affinityArbiter | indent 8 }}
       {{- end -}}
-      {{- if .Values.nodeSelector }}
+      {{- if (or .Values.nodeSelector .Values.nodeSelectorArbiter) }}
       nodeSelector:
-{{ toYaml .Values.nodeSelector | indent 8 }}
+{{ toYaml (default .Values.nodeSelector .Values.nodeSelectorArbiter) | indent 8 }}
```

Comment:
Please, user `nindent` instead of `indent`.

```yaml
nodeSelector: {{ toYaml (default .Values.nodeSelector .Values.nodeSelectorArbiter) | nindent 8 }}
```

Reply:
No problem, I change it :)